test/model/src/driver/kv_workload_runner_wt.cpp

```diff
@@ -149,7 +149,8 @@ kv_workload_runner_wt::run(const kv_workload &workload)
                         _state->expect_crash = true;
                         _state->crash_index = p;
                     }
-                    run_operation(op);
+                    _state->num_operations = p + 1;
```

Comment:
Since p is not declared in the for() this line could be moved after the loop, unless it is possible to exit the loop prematurely via an exception. If moved the line would be different since p is already incremented: _state->num_operations = p;